aroma/features.py

```diff
@@ -198,55 +202,26 @@ def feature_spatial(fslDir, tempDir, aromaDir, melIC):
         # Get sum of Z-values within the total Z-map (calculate via the mean
         # and number of non-zero voxels)
         tempICdata = tempIC.get_fdata()
-        totVox = np.sum(tempICdata != 0)  # number of nonzero voxels in image
+        totSum = np.sum(tempICdata)
```

Comment:
They count the number of non-zero voxels while you're suggesting we sum the Z-values (as the comment says). Do we know which one is correct? 

Evey other sum variable is also calculated by counting the number of non-zero values.

Reply:
They calculate the sum by counting the non-zero voxels, then calculating the mean across non-zero voxels, and the multiplying the number of non-zero voxels by the mean of the non-zero voxels. The results should be the same as just summing all voxels.

Reply:
Then why don't we just sum all non-zero voxels? It sounds simpler to me.

Reply:
Zeros don't contribute to a sum though, so filtering the voxels based on that doesn't end up changing anything. I also think that `np.sum(tempICdata)` is simpler than `np.sum(tempICdata[tempICdata != 0])`.

Reply:
I understand now, thanks!